xonsh/environ.py

```diff
@@ -1261,10 +1261,8 @@ def xonshrc_context(rcfiles=None, execer=None, ctx=None):
     return ctx
 
 
-def windows_env_fixes(ctx):
+def windows_foreign_env_fixes(ctx):
     """Environment fixes for Windows. Operates in-place."""
-    # Windows default prompt doesn't work.
-    ctx['PROMPT'] = DEFAULT_PROMPT
     # remove these bash variables which only cause problems.
     for ev in ['HOME', 'OLDPWD']:
```

Comment:
There's an argument on Windows that you actually want foreign shell import to default to off. They may have a cygwin/bash or git/bash installation on the PATH that isn't much used and importing stuff from it may have some surprising results. It's different than on Linux where your bash setup is what you've probably been living with.

This is a whole separate discussion though, and I haven't been living with the foreign shell stuff enough to have a strong opinion. Just thought I'd note it as a 'first impressions of a Windows user'.